docs-gb/operational-monitoring/observability.md

```diff
@@ -0,0 +1,128 @@
+---
+description: >-
+  Installing kube-prometheus-stack in the same Kubernetes cluster that hosts the
+  Seldon Core 2.
+---
+
+# Monitoring
+
+`kube-prometheus`, also known as Prometheus Operator, is a popular open-source project that provides complete monitoring and alerting solutions for Kubernetes clusters. It combines tools and components to create a monitoring stack for Kubernetes environments.
+
+{% hint style="info" %}
+**Note**: Always install Prometheus within the same Kubernetes cluster as the Seldon Core 2.
```

Comment:
Is there a reason we specify this constraint? Core 2 should run just fine with an externally-managed Prometheus instance.

Reply:
Hi @lc525  and @paulb-seldon  incorporated the changes that you suggested. Please review and state your approval.